smarts/core/default_map_builder.py

```diff
@@ -23,11 +23,61 @@
 from typing import NamedTuple, Tuple
 from pathlib import Path
 from smarts.core.road_map import RoadMap
-from smarts.core.utils.file import file_md5_hash
+from smarts.core.utils.file import file_md5_hash, path2hash
+
 
 _existing_map = None
 
 
+def _cache_result(map_spec, road_map, road_map_hash: str):
+    global _existing_map
+    from smarts.sstudio.types import MapSpec
+
+    class _RoadMapInfo(NamedTuple):
+        map_spec: MapSpec
+        obj: RoadMap
+        map_hash: str
+
+    _existing_map = _RoadMapInfo(map_spec, road_map, road_map_hash)
+
+
+def _clear_cache():
+    global _existing_map
+    if _existing_map:
+        import gc
+
+        # Try to only keep one map around at a time...
+        del _existing_map
+        _existing_map = None
+        gc.collect
```

Comment:
```suggestion
        gc.collect()
```

Reply:
good catch.  I typed that in instead of copy/pasting like I should have done.  will fix.